### PR TITLE
Add links to presentations to Readme

### PR DESCRIPTION
**Pull request by rugk**
I guess this is a good resource/cross-link to the presentations, so people can look it up.

I needed to find it myself manually, because I remembered there was such a thing.

### CODE REVIEW

**Comment by darkdragon-001** [comment]
Looks like a nice addition. Just one thing about the order? Could you move it before _Legal Stuff_ and _Contact_ as those two should be the last ones imho?

File: README.md [hunk 1]
```diff
@@ -181,3 +188,10 @@ Contact
 
 Twitter: [@bundesgit](https://twitter.com/bundesgit)
 
+
+More information
```
Comment (darkdragon-001) [inline README.md:192]: Maybe "References" is a more suitable title?

```suggestion
References
```

Comment (mk-pmb) [reply README.md:192]: To me, "References" seems to mean something authoritative. How about just "See also"?

Comment (darkdragon-001) [reply README.md:192]: I thought about something like _press coverage_? Maybe you are right that this is not really _outside press_ (someone else) but more _inside presentation_.
So I am fine with the other suggestions as well.

Comment (rugk) [reply README.md:192]:
```suggestion
See also
```

File: README.md [hunk 2]
```diff
@@ -79,6 +79,14 @@ bei der [Ausarbeitung eines Git-Workflows](https://github.com/bundestag/gesetze/
 für dieses Repostiory gebeten.
 
 
+Mehr Informationen
```
Comment (darkdragon-001) [inline README.md:82]: We should keepboth languages consistent:

```suggestion
Siehe auch
```

**Review by darkdragon-001 — CHANGES_REQUESTED** [review]

**Comment by darkdragon-001** [comment]
Lgtm

@mk-pmb if you don't have further. comments please merge!

**Review by darkdragon-001 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -170,6 +177,14 @@ Please help shape a [Git Workflow](https://github.com/bundestag/gesetze/wiki/Git
 for this repository.
 
 
+See also
+----------------
+
+This project was also presented or mentioned at various conferences:
+* [chmod o+rw bundestag - More transparency and participation in the legislative process](https://media.ccc.de/v/29c3-5263-de-en-mehr_transparenz_und_teilhabe_im_gesetzgebungsprozess_h264) (German), 29c3
+* [Democracy DevOps](https://media.ccc.de/v/froscon2019-2510-demokratie-devops) (German), FrOSCon 2019
```
Comment (patriciapaluna186-dot) [inline README.md]: Kalender

**Pull request merged**